README.md

```diff
@@ -37,12 +37,18 @@ MacVim.
 
 How do you do it?
 
-#### OS X 10.9 (Mavericks)
+#### OS X 10.9 (Mavericks) with boxen-web
 
 If you are using [`b26abd0` of boxen-web](https://github.com/boxen/boxen-web/commit/b26abd0d681129eba0b5f46ed43110d873d8fdc2)
-or newer, it will be automatically installed as part of Boxen.
+or newer, Xcode Command Line Tools will be automatically installed as part of Boxen.
 Otherwise, follow instructions below.
 
+##### OS X < 10.9 (Mavericks) without auto install
+
+1. Install Xcode from the Mac App Store.
+1. In terminal, run `xcode-select --install`
```

Comment:
In Homebrew-land we support the current version and two prior so that feels like a reasonable approach here. I think even after that point it's still worth landing PRs to fix 10.8 and below but arguably can tweak the docs etc. to make them less clunky then.